ext/jsonnet/extconf.rb

```diff
@@ -13,8 +13,8 @@ def using_system_libraries?
   require 'mini_portile2'
   message "Using mini_portile version #{MiniPortile::VERSION}\n"
 
-  recipe = MiniPortile.new('jsonnet', 'v0.10.0')
-  recipe.files = ['https://github.com/google/jsonnet/archive/v0.10.0.tar.gz']
+  recipe = MiniPortile.new('jsonnet', 'v0.13.0')
```

Comment:
As @UltimateGeek mentioned, you might also want to upgrade this to 0.16.0.

Again, sorry for my late reply.

Reply:
No worries at all, thank you for replying.

That's a great idea, I just bumped the version to 0.16.0 5e43ae3. All tests passed locally.